sharktank/sharktank/utils/iree.py

```diff
@@ -30,6 +30,8 @@
 )
 from sharktank.utils import verify_exactly_one_is_not_none
 from .tree import Tree
+from .logging import get_logger
+from .debugging import flags as debug_flags
```

Comment:
I think this import may interfere with the [import below](https://github.com/nod-ai/shark-ai/blob/944d32e3213324b9b9327b45133edaf063f7f3b7/sharktank/sharktank/utils/iree.py#L630) when constructing the callback.
Somehow [here](https://github.com/nod-ai/shark-ai/blob/944d32e3213324b9b9327b45133edaf063f7f3b7/sharktank/sharktank/utils/iree.py#L642) `debugging` is not defined. I guess it tries to run the import lazily when calling the callback and it fails.
This is the Python function that gets called by IREE during the module execution.
Quite wired.